exporters/trace/README.md

```diff
@@ -1 +1,47 @@
 # Cloud Trace Exporter for OpenTelemetry
+
+  OpenTelemetry Google Cloud Trace Exporter allows the user to send collected traces to Google Cloud. 
+  
+  <a href="https://cloud.google.com/trace">Google Cloud Trace</a> is a distributed tracing backend system. It helps developers to gather timing data needed to troubleshoot latency problems in microservice & monolithic architectures. It manages both the collection and lookup of gathered trace data.
+
+## Setup
+  Google Cloud Trace is a managed service provided by Google Cloud Platform.
+
+## Installation
+  Not currently ready as a package to be imported. To use presently, one would have to clone this GitHub repo.
+## Usage
+  If you are running in a GCP environment, the exporter will automatically authenticate using the environment's service account. If not, you will need to follow the instructions in Authentication.  
+    
+  The TraceExporter constructor takes in three parameters: String for projectId, TraceServiceClient to create a traceServiceClient instance, and Map<String, AttributeValue> for the fixed attributes of a span.  
+  So, we need to import the following: 
+  ```
+  import com.google.cloud.trace.v2.TraceServiceClient;
+import com.google.devtools.cloudtrace.v2.AttributeValue;
+import com.google.devtools.cloudtrace.v2.ProjectName;
+import com.google.devtools.cloudtrace.v2.Span;
+```
+  Declare and initialize the variables that will be used in the constructor parameters.  
+  Then, we can create a TraceExporter with, for example:
+  ```
+  TraceExporter exporter = new TraceExporter(projectId, traceServiceClient, fixedAttributes);
+  ```
+  Start tracing and collecting SpanData. To export these spans, use the export method, where spanDataList is a Collection of SpanData already collected:
+  ```
+  exporter.export(spanDataList)
+  ```
+
+## Authentication
+  This exporter uses <a href="https://github.com/googleapis/google-cloud-java">google-cloud-java</a>, for details about how to configure the authentication see <a href="https://github.com/googleapis/google-cloud-java#authentication">here</a>.  
```

Comment:
does it actually? I thought that you were going to add authentication in the future

Reply:
It does; authentication is done by generating a JSON service account key and then setting the GOOGLE_APPLICATION_CREDENTIALS environment variable to the location of that key. This process is referenced in the hyperlink included in the word "here" (this link: https://github.com/googleapis/google-cloud-java#authentication)

Reply:
### Also, because I can't reply to the "screenshot" comment, I'll leave the screenshot of the updated readme here:

![Screenshot 2020-07-08 at 11 29 37 AM](https://user-images.githubusercontent.com/43458707/86957097-0de4f380-c10f-11ea-8a97-fb749b5812e1.png)